### PR TITLE
Replace instances for OSSEC Alert Key

### LINKED ISSUE #44: Define and standardize OSSEC Alert Key terminology
Similar to freedomofpress/securedrop#4122, let's standardize how we describe the keypair used for OSSEC alerts, and describe this in the glossary. For example, we sometimes call it "the GPG key that OSSEC will encrypt alerts to", but in the installation guide we also refer to it as "the admin’s GPG public key file (for encrypting OSSEC alerts)". Repeating the same long definition over and over again risks introducing small inconsistencies like that.

I would recommend that we call it the *OSSEC Alert Key* or *OSSEC Alert Public Key*, and that we call the key for the 24h alerts the *Journalist Alert Key* or *Journalist Alert Public Key* (use the most precise qualifier -- public or private -- where possible).

### PR DESCRIPTION
**Pull request by joaedwar**
## Status

Ready for Review

## Description of Changes

* **Description**: Define and standardize OSSEC Alert Key terminology. Replace "the admin’s GPG public key file (for encrypting OSSEC alerts)" and similar description with OSSEC Alert Key or OSSEC Alert Public Key, and call the key for the 24h alerts the Journalist Alert Key or Journalist Alert Public Key.

* Fixes #44

## Testing

*

## Release 

* 

## Checklist (Optional)

- [ ] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [ ] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/install.rst
Comment (eloquence) [inline docs/install.rst:78]: Per our standard convention, we should italicize this term and add a definition to the glossary (happy to talk through what this key is used for).

**Review by eloquence — CHANGES_REQUESTED** [review]
Thanks @joaedwar! See inline comment; the biggest requested change is to add a glossary entry. Note that you currently have a lint failure.

**Comment by joaedwar** [comment]
@eloquence

File: docs/glossary.rst
Comment (eloquence) [inline docs/glossary.rst:54]: is an alert -> is used for encrypting the alert

File: docs/glossary.rst
Comment (eloquence) [inline docs/glossary.rst:57]: We should not use absolute URLs to link to other chapters of the documentation. reStructured text supports the `:doc:` and `:ref:` syntax to cross-reference other chapters and sections, respectively. The glossary chapter has other absolute URL references which should be replaced with `:doc`: or `:ref:`. If you want to take a stab at cleaning up the links in this chapter, please add a commit to do so, otherwise I can do it before merging.

File: docs/glossary.rst
Comment (eloquence) [inline docs/glossary.rst:52]: If we introduce this term newly to the glossary, we should make sure we do in fact use it in the documentation, i.e. the chapter about journalist alerts: https://docs.securedrop.org/en/stable/journalist.html#daily-journalist-alerts-about-submissions

File: docs/threat_model/mitigations.rst
Comment (eloquence) [inline docs/threat_model/mitigations.rst:193]: This line refers to the email account credentials used to receive emails, not the keys used to encrypt them. So the original language is correct here, and should not be modified.

File: docs/threat_model/mitigations.rst
Comment (eloquence) [inline docs/threat_model/mitigations.rst:195]: In both cases the *private* key is what we need to worry about most; the *public* key will not allow an adversary to read encrypted emails.

File: docs/set_up_admin_tails.rst
Comment (eloquence) [inline docs/set_up_admin_tails.rst:207]: Let's italicize here as well.

File: docs/glossary.rst
Comment (eloquence) [inline docs/glossary.rst:103]: These instructions do not belong in the glossary.

**Review by eloquence — CHANGES_REQUESTED** [review]
Thanks for your continued work on this, @joaedwar. Still a bit more cleanup required, happy to go over this in more detail in our 1:1 :)

**Review by eloquence — APPROVED** [review]
Thanks @joaedwar, fixes look good! Rebased & made some final minor tweaks, approving with those changes.

**Pull request merged**
